new/LIT7101QeneAmh.xml

```diff
@@ -0,0 +1,100 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7101QeneAmh" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="am" xml:id="t1">እየደረስብኝ፡ ነው፡ በሰኔ፡ ፈረሰኛው፡ ሞት፡ ነፍሴን፡ ሳልጠቅማት፡ ወየው፡ ሰዎች፡ እያሉ፡ አዝማሪ፤</title>
```

Comment:
As you know, I am not an expert on Amharic phonetics and morphology. The change from ኢደርስበኝ፡ to እየደረሰብኝ፡ seems to me to be a strong emendation. Can the different spellings be seen as archaic or perhaps dialectal variations, or rather as scribal errors or less significant orthographic variations? In the first case, it would be reasonable to keep the archaic or dialectal form in the title.

Reply:
The same for ኢያሉ፡ / እያሉ፡

Reply:
I am also not really competent here. What do you think, @DenisNosnitsin1970 ?

Reply:
do I understand you correctly, @DenisNosnitsin1970 , that it would be best to encode `<sic>ኢደርስ<choice><sic>በ</sic><corr>ብ</corr></choice>ኝ፡</sic>` ?